app/templates/search-details.html

```diff
@@ -67,6 +67,29 @@
             </div>
           </div>
 
+          <div class="panel panel-default panel-search">
+            <div class="panel-heading panel-collapse-header" data-toggle="collapse" data-parent="#template-versions" href="#template-versions" aria-expanded="true" aria-controls="template-collapse">
+              Versions
+            </div>
+            <div class="panel-body panel-collapse collapse" role="tabpanel" aria-labelledby="template-versions" id="template-versions">
+              <div class="form-group" v-for="(key, values) in filterVersions(versions)">
+                <select
+                   v-model="selectedVersions[key]"
+                   id="{{ key }}"
```

Comment:
`{{ }}` is not allowed in attributes starting from version 2.0, so please use `v-bind:id="key"`.